src/main/java/design/MazeDesignerInteractor.java

```diff
@@ -0,0 +1,47 @@
+package design;
+
+import entities.DesignableMaze;
+
+public class MazeDesignerInteractor implements MazeDesignerInputBoundary {
+
+    DesignableMaze dm;
```

Comment:
Make this a private instance attribute and refer to [this comment](https://github.com/CSC207-2022F-UofT/course-project-wasted-potential/pull/16/files#r1028497837)
`private DesignableMazde dm;`

Also add instance attributes:
`private RandomizedPrim mg;`
`private MazeDesignerOutputBoundary ob`